wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -821,6 +821,16 @@ impl super::InstanceShared {
                 let mut_ref = features.robustness2.as_mut().unwrap();
                 mut_ref.p_next = mem::replace(&mut features2.p_next, mut_ref as *mut _ as *mut _);
             }
+            if capabilities.supports_extension(vk::ExtDepthClipEnableFn::name()) {
+                features.depth_clip_enable = Some(
+                    vk::PhysicalDeviceDepthClipEnableFeaturesEXT::builder()
+                        .depth_clip_enable(true)
```

Comment:
this is not a place where features are enabled. This is where they are queried, so this line needs to be removed (see like everything else is done in this block)

Reply:
Do I just remove that line? I can't seem to find where the features are enabled

Reply:
Yes. It's enabled here - https://github.com/gfx-rs/wgpu/blob/943145b4df9dd4c0bc236a76ef96acd13c46fead/wgpu-hal/src/vulkan/adapter.rs#L305

Reply:
Oh okay, thanks for pointing me to the right place.

Reply:
still unresolved

Reply:
classic forgot to push